frontend/src/pages/admin/Admin.tsx

```diff
@@ -0,0 +1,196 @@
+import { Container, Card } from '../../components';
+import React, { Component, useEffect } from 'react';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Button, MenuItem, Select, TextField } from '@material-ui/core';
+import jwtDecode, { JwtHeader } from 'jwt-decode';
+import './Admin.scss';
+
+export default function Admin() {
+  interface User {
+    userID: number;
+    name: string;
+    role: string;
+    email: string;
+  }
+
+  const [accountType, setAccountType] = useState<string>('');
+
+  const onAccountTypeChange = (e: React.ChangeEvent) => {
+    const select = e.target as HTMLSelectElement;
+    const type = select.value;
+    setAccountType(type);
+  };
+
+  const changeAccountType = (user: User) => async (e: React.ChangeEvent) => {
+    const select = e.target as HTMLSelectElement;
+    const type = select.value;
+    setAccountType(type);
+
+    const request = await fetch('http://localhost:5000/user/' + user.userID, {
+      method: 'PUT',
+      headers: {
+        Authorization: `bearer ${localStorage.token}`,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ userID: user.userID, name: user.name, role: type, email: user.email })
+    });
+    //const response = await request.json();
```

Comment:
Dead code to be removed?

Reply:
We'll mark it as a bug to be fixed next sprint.

Reply:
oopsies